onnxruntime/core/graph/graph_utils.h

```diff
@@ -69,6 +69,10 @@ bool AllNodeInputsAreConstant(const Graph& graph, const Node& node, InitializedT
 /** Gets the name of the incoming NodeArg with the specified index for the given node. */
 const std::string& GetNodeInputName(const Node& node, int index);
 
+/** Gets the index of an input arg with the specified input arg name.
+**/
```

Comment:
nit - make it one line if you are not adding more comments, like params/return, etc.